overlord/install/install.go

```diff
@@ -253,11 +253,6 @@ func PrepareEncryptedSystemData(model *asserts.Model, keyForRole map[string]keys
 	// make note of the encryption keys
```

Comment:
does the doc comment of this function need to explain the new responsability of the caller?

Reply:
I have added in the doc of this function